api/gen/v1alpha/core.swagger.yaml

```diff
@@ -158,9 +158,15 @@ paths:
       operationId: LicenseService_GetSeats
       responses:
         "200":
-          description: A successful response.
+          description: A successful response.(streaming responses)
```

Comment:
Do we need to send content type like application/x-ndjson or application/stream+json